docs/content/en/docs/implementing/evaluatemetrics.md

```diff
@@ -95,14 +100,82 @@ spec:
     key: DT_TOKEN
 ```
 
-## Accessing Metrics via the Kubernetes Custom Metrics API
+### Define KeptnMetric information
+
+The [KeptnMetric](../yaml-crd-ref/metric.md) resource
+defines the information you want to gather,
+specified as a query for the particular observability platform
+you are using.
+You can define any type of metric from any data source.
+
+In our example, we define two bits of information to retrieve:
+
+* Number of CPUs, derived from the `dev-prometheus` data platform
+* `availability` SLO, derived from the `dev-dynatrace` data platform
 
-`KeptnMetrics` can also be retrieved via the Kubernetes Custom Metrics API.
+Each of these are configured to fetch data every 10 seconds
+but you could configure a different `fetchIntervalSeconds` value
+for each metric.
 
-### Retrieve KeptnMetric values with kubectl
+The
+[keptn-metric.yaml](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/keptn-metric.yaml)
+file for our example looks like:
+
+```yaml
+apiVersion: metrics.keptn.sh/v1alpha2
+kind: Keptnmetric
+metadata:
+  name: available-cpus
+  namespace: simplenode-dev
+spec:
+  provider:
+    name: dev-prometheus
+  query: "sum(kube_node_status_cvapacity{resources`cpu`})"
+  fetchIntervalSeconds: 10
+---
+apiVersion: metrics.keptn.sh/v1alpha2
+kind: Keptnmetric
+metadata:
+  name: availability-slo
+  namespace: simplenode-dev
+spec:
+  provider:
+    name: dev-dynatrace
+  query: "func:slo.availability_simplenodeservice"
+  fetchIntervalSeconds: 10
+```
+
+Note the following:
+
+* Populate one YAML file per metric
+  then apply all of them.
+* Each metric is assigned a unique `name`.
```

Comment:
how many resources you define in one YAML file doesn't matter at all
```suggestion
* Each metric is assigned a unique `name`.
```

Reply:
Interesting.  I forget who recommended that.